README.md

```diff
@@ -94,13 +94,6 @@ branches are Malte Helmert, Gabi Röger and Silvia Richter.
 
 ## License
 
-The following directory is not part of Fast Downward as covered by
```

Comment:
Note to reviewers: we still have code in `ext/optional.hh` but this is public domain code, we don't have to mention it here.